girder/api/rest.py

```diff
@@ -740,7 +741,9 @@ def _setCommonCORSHeaders():
         # If there is no origin header, this is not a cross origin request
         return
 
-    allowed = Setting().get(SettingKey.CORS_ALLOW_ORIGIN)
+    allowed = os.getenv('GIRDER_CORS_ALLOW_ORIGIN')
```

Comment:
This logic is now obviated by #3451

Reply:
I've rebased to now use your general code path to set this setting and backed out my change for this one setting.